geomesa-feature/pom.xml

```diff
@@ -34,6 +34,11 @@
 
         <!-- Test dependencies -->
         <dependency>
+            <groupId>com.esotericsoftware</groupId>
+            <artifactId>kryo</artifactId>
+            <version>3.0.0</version>
```

Comment:
Two questions:
1.  Does anything else in sight use Kryo?  Specifically, what about our listed Spark version?  

2.  Do we need a new CQ for this dependency?